aggregating-profiler-jmh/build.gradle.kts

```diff
@@ -19,16 +19,16 @@ dependencies {
     /**
      * Runtime
      */
-    compile(Libs.slf4j_api)
-    compile(Libs.kotlin_jdk8)
-    compile(Libs.kotlin_stdlib)
-    compile(Libs.kotlin_reflect)
-    compile(Libs.slf4j_simple)
-    compile(Libs.jmh)
-    compile(Libs.jmhGeneratorAnn)
-    compile(Libs.jmhGeneratorBytecode)
-
-    compile(project(":aggregating-profiler"))
+    implementation(Libs.slf4j_api)
+    implementation(Libs.kotlin_jdk8)
+    implementation(Libs.kotlin_stdlib)
+    implementation(Libs.kotlin_reflect)
+    implementation(Libs.slf4j_simple)
+    implementation(Libs.jmh)
+    implementation(Libs.jmhGeneratorAnn)
+    implementation(Libs.jmhGeneratorBytecode)
+
+    implementation(project(":aggregating-profiler"))
```

Comment:
as far as I understand, JMH module is used to test profiler's perfomance, so it shouldn't be imported as a dependency. Therefore i used only `implementation` configuration here